<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abalone</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="toolButtonServeur">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnexion">
        <property name="text">
         <string>Rejoindre une partie</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelAdversaire">
          <property name="styleSheet">
           <string notr="true">color:white;background-color:blue;</string>
          </property>
          <property name="text">
           <string>Bleus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberBleu"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="BoutonDirection" name="pushButtonSE">
            <property name="text">
             <string>SudEst</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="BoutonDirection" name="pushButtonE">
            <property name="text">
             <string>Est</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="BoutonDirection" name="pushButtonSO">
            <property name="text">
             <string>SudOuest</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="BoutonDirection" name="pushButtonNE">
            <property name="text">
             <string>NordEst</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="BoutonDirection" name="pushButtonO">
            <property name="text">
             <string>Ouest</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="BoutonDirection" name="pushButtonNO">
            <property name="text">
             <string>NordOuest</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelJoueurLocal">
          <property name="text">
           <string>Rouges</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberRouge"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlanceServeur">
   <property name="text">
    <string>lanceServeur</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoutonDirection</class>
   <extends>QPushButton</extends>
   <header>boutondirection.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
